### PR TITLE
Documentation and installer helpers

### PR DESCRIPTION
**Pull request by aethelwulffe**
Re-committing the changes to the documentation directory structure
posted to the old repo.
This commit organizes the documentation directory, and adds the
installer helper docs to assets.  The new files are utilized by the
XAMPP package install helper, and the compiled installer itself.  An
attempt was made to replace 1:1 files existing in the current master and
overwrite them into the new structure over the versions used in the
original branch last September.

### CODE REVIEW

**Comment by aethelwulffe** [comment]
See #448 and #449 for related activity.

**Comment by aethelwulffe** [comment]
I had to add a second phpgacl readme to resolve the conflicts.  Playing on top was broken for some weird reason.  I will push a second pull request to remove the excess file.  Please, pretty pretty pretty pretty please merge this stuff so the XAMPP package stuff can be built without me having to hand-load every piece every time.

File: Documentation/README-phpgacl.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+             Hints for Using phpGACL with LibreHealth EHR
+             by Rod Roark <rod at sunsetsystems dot com>
+
+Installation Instructions
+
```
Comment (teryhill) [inline Documentation/README-phpgacl.md]: Is there still a manual install of this

File: Documentation/README-phpgacl.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+Upgrading Instructions
+
+After you have upgraded to a new version of LibreHealth EHR, you should
+run the acl_upgrade.php program using your web browser
+(e.g. http://libreehr.location/acl_upgrade.php). This will ensure your
+phpGACL database contains all the required LibreHealth EHR Access Control
+Objects.
+
```
Comment (teryhill) [inline Documentation/README-phpgacl.md]: isn't this part of the install script?

File: Documentation/README-phpgacl.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+1. Header notes of the library/acl.inc file
+2. acl_setup.php file
+3. acl_upgrade.php file
+
+
+Miscellaneous Information
+(the below information is only applicable
+to LibreHealth EHR versions less than 2.9.0.3 or to users who choose to
+install an external version of phpGACL)
+
```
Comment (teryhill) [inline Documentation/README-phpgacl.md]: No version 2.9.0.3 Should this section be dumped?

File: Documentation/README-phpgacl.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+has not been automatically installed.  Setting it up takes some careful
+study, planning and a bit of time.  If you don't have the time then you
+should hire an experienced person to set things up for you.  Helpful
+installation and configuration instructions can be found on the wiki at the
+librehealth.io site.
+
+Alternatively, it's possible to set up your own access rules without
+using phpGACL by customizing the code in library/acl.inc.  See that
+module for more information.
+
```
Comment (teryhill) [inline Documentation/README-phpgacl.md]: Don't think the above section is needed

**Comment by aethelwulffe** [comment]
All good points, but that stuff is not within the scope of this commit.
This is a re-organization of the files so that the actual editing of them can be organized.  We have TWO people waiting in the wings (One that both of us know) and two new folks here that are trying to update this stuff.  We need to get the receipts all in one trash can before we hand them over to the accountant.
More importantly, this commit handles the stuff for the XAMPP installer...and provides additional resources for additional platforms to use in their documentation.

**Comment by tmccormi** [comment]
I'm not clear if this is ready to merge or not? @aethelwulffe

**Comment by aethelwulffe** [comment]
yes, absolutely.  Terry has some points on the stuff in the docs, but this is just organizing the docs and adding the installer stuff (yes, with some improvements to the originals).  I need this so that I can slipstream the XAMPP packs without having to do a full build of all the little pieces everywhere each time.

**Pull request merged**